bundle/src/main/java/com/jfrog/ide/eclipse/scan/NpmScanManager.java

```diff
@@ -29,6 +30,12 @@ void refreshDependencies(IProgressMonitor monitor) throws IOException {
 
 	@Override
 	void buildTree() throws CoreException, JsonProcessingException, IOException {
-		setScanResults(npmTreeBuilder.buildTree(getLog()));
+		try {
+			setScanResults(npmTreeBuilder.buildTree(getLog(), false));
```

Comment:
is `toast` not avaliable in eclipse? if so add comment so we will know

Reply:
In this plugin we implemented a logger. so the errors are presented in the problems tab